UnityProject/Assets/Scripts/Tilemaps/Behaviours/Meta/Atmospherics/Data/GasMix.cs

```diff
@@ -438,6 +438,17 @@ public void MergeGasMixes(List<PipeData> otherGas)
 			}
 		}
 
+		public GasSO GetBiggestGasSOInMix()
+		{
+			GasSO bigGas = null;
+			foreach(var gas in GasData.GasesArray)
+			{
+				if (bigGas == null || gas.Moles < bigGas) continue;
```

Comment:
```suggestion
				if (bigGas != null && gas.Moles < bigGas.Moles) continue;
```

Sorry my bad, should be this conditional. 

You also need to put `bigGas.Moles` not just `bigGas` as that just gets the gas index,